addons/mail/static/src/discuss/core/web/channel_member_patch.js

```diff
@@ -26,6 +28,7 @@ patch(ChannelMember.prototype, {
                 channelMember: this.member,
             });
         }
+        this.state.isAvatarCardOpen = true;
```

Comment:
Only necessary inside the `if`.

But `avatarCard.isOpen` needs to be adapted there too?